applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -160,17 +176,22 @@ int bt_le_audio_tx_send(struct bt_bap_stream **bap_streams, struct le_audio_enco
 		return -EINVAL;
 	}
 
-	/* When sending data, we always send ts = 0 to the first active transmitting channel.
+	/* When sending ISO data, we always send ts = 0 to the first active transmitting channel.
 	 * The controller will populate with a ts which is fetched using bt_iso_chan_get_tx_sync.
 	 * This timestamp will be submitted to all the other channels in order to place data on all
 	 * channels in the same ISO interval.
 	 */
 
-	uint32_t ts_common = 0;
+	uint32_t common_tx_sync_ts_us = 0;
+	uint32_t curr_time_ts_us = 0;
 	bool ts_common_acquired = false;
 	uint32_t common_interval = 0;
 
 	for (int i = 0; i < streams_to_tx; i++) {
+		if (tx_info_arr[i].iso_tx.seq_num == 0) {
```

Comment:
Is this the rollover-guarding? Maybe a comment would be good

Reply:
Changed.